test/e2e_node/resource_metrics_test.go

```diff
@@ -64,13 +63,13 @@ var _ = framework.KubeDescribe("ResourceMetricsAPI", func() {
 			ginkgo.By("Waiting 15 seconds for cAdvisor to collect 2 stats points")
 			time.Sleep(15 * time.Second)
 		})
-		ginkgo.It("should report resource usage through the v1alpha1 resouce metrics api", func() {
+		ginkgo.It("should report resource usage through the resource metrics api", func() {
 			ginkgo.By("Fetching node so we can know proper node memory bounds for unconstrained cgroups")
 			node := getLocalNode(f)
 			memoryCapacity := node.Status.Capacity["memory"]
 			memoryLimit := memoryCapacity.Value()
```

Comment:
@dashpole can you help me understand this comment? It looks like it refers to the following couple of lines, and then goes into the matcher. I'm guessing that this is sort of a default return value of the node having access to all of it's own memory?

Reply:
The matcher checks that the returned value is within a range.  Memory usage should not exceed the capacity of the node, so it is an upper bound on some ranges IIRC.